components/keyrings/common/data/global_default_mr.h

```diff
@@ -19,7 +19,7 @@
 
 inline ::boost::container::pmr::memory_resource
     * ::boost::container::pmr::get_default_resource() BOOST_NOEXCEPT {
-  psi_memory_resource *global_default_mr = new psi_memory_resource{};
+  static psi_memory_resource *global_default_mr = new psi_memory_resource{};
```

Comment:
Please, why not? On the other, is there not an already available `ps_memory_resource` published in another service?

```
psi_memory_resource global_default_mr;

inline ::boost::container::pmr::memory_resource
    * ::boost::container::pmr::get_default_resource() BOOST_NOEXCEPT {
  return &global_default_mr;
}
```

Reply:
Please check PS-8004, as this leak is caused by the fix for that issue (the original code, before the fix was exactly what you suggested here].